allennlp/predictors/predictor.py

```diff
@@ -96,6 +179,16 @@ def predict_instance(self, instance: Instance) -> JsonDict:
         outputs = self._model.forward_on_instance(instance)
         return sanitize(outputs)
 
+    def predictions_to_labeled_instances(self,
+                                         instance: Instance,
+                                         outputs: Dict[str, np.ndarray]) -> List[Instance]: \
+                                         # pylint: disable=unused-argument,no-self-use
```

Comment:
Remove the backslash, put this pylint command inside the method body, either under or just above the docstring (maybe under?).